spring/src/test/java/org/axonframework/spring/eventhandling/deadletter/SpringJpaDeadLetteringIntegrationTest.java

```diff
@@ -16,13 +16,13 @@
 
 package org.axonframework.spring.eventhandling.deadletter;
 
-import org.axonframework.common.jpa.EntityManagerProvider;
-import org.axonframework.common.jpa.SimpleEntityManagerProvider;
+import org.axonframework.common.legacy_jpa.EntityManagerProvider;
+import org.axonframework.common.legacy_jpa.SimpleEntityManagerProvider;
 import org.axonframework.common.transaction.TransactionManager;
 import org.axonframework.eventhandling.EventMessage;
 import org.axonframework.eventhandling.deadletter.DeadLetteringEventHandlerInvoker;
 import org.axonframework.eventhandling.deadletter.DeadLetteringEventIntegrationTest;
-import org.axonframework.eventhandling.deadletter.jpa.JpaSequencedDeadLetterQueue;
+import org.axonframework.eventhandling.deadletter.legacy_jpa.JpaSequencedDeadLetterQueue;
```

Comment:
I expect this to be run with Jakarta

Reply:
Added module with tests run against hibernate 6, using the jakarta classes.